dataprofiler/profilers/profiler_options.py

```diff
@@ -138,6 +143,26 @@ def validate(self, raise_error: bool = True) -> list[str] | None:
             return errors
         return None
 
+    @classmethod
+    def load_from_dict(cls, data) -> BaseOption:
+        """
+        Parse attribute from json dictionary into self.
+
+        :param data: dictionary with attributes and values.
+        :type data: dict[string, Any]
+
+        :return: Profiler with attributes populated.
+        :rtype: BaseColumnProfiler
+        """
+        profile = cls()
+
+        for attr, value in data.items():
+            if isinstance(value, dict) and "class" in value:
```

Comment:
is it safe to assume that sub-object would be an options object under these conditions? I think its safe to assume now, but for future reference?